docs/pubsub-usage.rst

```diff
@@ -213,13 +228,13 @@ Fetch pending messages for a pull subscription:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> from gcloud.pubsub import Subscription
-   >>> topic = Topic('topic_name')
-   >>> subscription = Subscription('subscription_name', topic)
-   >>> with topic:
-   ...     topic.publish('this is the first message_payload')
-   ...     topic.publish('this is the second message_payload',
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
+   >>> subscription = pubsub.Subscription('subscription_name', topic)
+   >>> with topic.batch() as batch:
+   ...     batch.publish('this is the first message_payload')
+   ...     batch.publish('this is the second message_payload',
    ...                   attr1='value1', attr2='value2')
    >>> received = subscription.pull()  # API request
```

Comment:
@tseaver WDYT?

PS @jgeewax we'd use PEP8 method names `get_ack_id` etc. (context switching from `node` to `python` can be difficult, I know)